src/cheri_sys_regs.sail

```diff
@@ -65,22 +65,33 @@
 /* Capability control csr */
 
 bitfield ccsr : xlenbits = {
+  ugclg   :  3,      /* Global Cap Load Gen bit for User pages */
+  sgclg   :  2,      /* Global Cap Load Gen bit for System (not User) pages */
   d       :  1,      /* dirty  */
   e       :  0       /* enable */
 }
 
-register mccsr : ccsr
-register sccsr : ccsr
+register sccsr : ccsr /* mccsr aliases sccsr */
 register uccsr : ccsr
```

Comment:
Bit weird to still have uccsr... I kinda get why, but that's primarily because uccsr currently serves no purpose. Also, if mccsr and sccsr alias then normally you'd have the definition be mccsr and sccsr be an alias for (or view into) it, not the other way round.

Reply:
Well, it "serves no purpose" because the `d` and `e` bits are not meaningfully implemented in sail, but they're still defined...

Reply:
Though maybe that does argue against making `sccsr` and `mccsr` alias.

Reply:
I'm not particularly happy with what I have just written in `cheri_pte.sail` -- "use `sccsr` if `haveSupMode()` and `mccsr` otherwise" -- but it might be viable.  I'm quite open to suggestions.

Reply:
@jrtc27 Opinions?